jenkins/bootstrap_test.py

```diff
@@ -1975,9 +1976,20 @@ def testNoBadVarsInJobs(self):
 
     def testValidJobEnvs(self):
         """Validate jobs/config.json."""
-        with open(bootstrap.test_infra('jobs/config.json')) as fp:
+        with open(bootstrap.test_infra('jobs/config.json')) as fp, open(bootstrap.test_infra('jobs/validOwners.json')) as ownfp:
             config = json.loads(fp.read())
+            validOwners = json.loads(ownfp.read())
             for job in config:
+                # onwership assertions
+                self.assertTrue('sigOwners' in config[job], job)
+                self.assertTrue(type(config[job]['sigOwners']) == list)
```

Comment:
this will produce unhelpful `False != True` error messages. Please use a more specific type check:

https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertIsInstance